src/main/java/org/opentripplanner/graph_builder/module/GtfsModule.java

```diff
@@ -264,7 +279,8 @@ private GtfsMutableRelationalDao loadBundle(GtfsBundle gtfsBundle) throws IOExce
 
     if (LOG.isDebugEnabled()) reader.addEntityHandler(counter);
 
-    for (Class<?> entityClass : reader.getEntityClasses()) {
+    var filteredEntities = getEntitiesToRead(reader);
+    for (Class<?> entityClass : filteredEntities) {
```

Comment:
It is not clear from these two lines what effect the `getEntitiesToRead` have. Maybe this is better:
```suggestion
    for (Class<?> entityClass : reader.getEntityClasses()) {
      if(skipEntityClass(entityClass)) {
        LOG.info("Skip entities: " + entityClass.getName());
        continue;
      }
```
Note that this preserve the abstraction level for the accessing the `reader` and logging. (I added the logging).